### ISSUE TITLE
MTGArena "Companion" Pawprint Icon

### ISSUE THREAD

**Issue by TripleEquals** [andrewgioia/keyrune#134]
In MTGArena there is an icon that is used specifically to identify companions. This icon would be useful for other applications that wish to denote companions. I'm unsure if it appears in any other colors. Below is a low res snapshot of the icon. The icon is just the pawprint, none of the surrounding border.
![CompanionSnip](https://user-images.githubusercontent.com/12373488/90344251-ce8eaa00-dfcc-11ea-9966-cd3c8b470b37.PNG)

**Comment by arjenbrandenburgh** [comment]
It feels that this may be better suited in @andrewgioia's other repository: https://github.com/andrewgioia/mana

**Comment by andrewgioia** [comment]
Yep I'll transfer this to Mana, thank you for the heads up @TripleEquals and @arjenbrandenburgh
